components/Carousel/Carousel.js

```diff
@@ -1,6 +1,39 @@
-class Carousel {
+  // let slider = document.querySelector('Carousel__item-focused')
+  let slides = document.querySelectorAll('Carousel__item');
+  let r = document.getElementById('R');
```

Comment:
It would be best to name these variable right/left, it is a bit more descriptive and helps others easily follow you code.